src/components/ReportDebugIds.tsx

```diff
@@ -40,6 +40,7 @@ export type ReportDebugIdsProps = {
   className?: string;
   displayStrings?: Partial<typeof defaultDisplayStrings>;
   children?: React.ReactNode;
+  hideBlankValues?: boolean;
```

Comment:
Instead of this prop, could we keep the same API by changing the behavior of the current data? Something like: If undefined, then hide the row. If empty string, then show "no value".

Reply:
Yeah, that's a better idea. I have implemented this.